### DISCUSSION TITLE
Jaeger collector.host in agent - not updating dynamically

### DISCUSSION THREAD

**Discussion by raz08** [jaegertracing/jaeger#2878] [General]
Hi Jaeger Community,

I tried updating collector.host from ConfigMap and these configmap values are consumed as environment variable in Jaeger agent deployment file (i.e. daemon set) . How to dynamically take changed collector.host and reload the config in Jager agent?

**Comment by jpkrohling** [comment]
Related to #2879, but in short: the operator doesn't currently take care of it, you'd have to manually trigger a rollout of the daemonset whenever the configmap changes. In the v2, we will likely have an automatic rollout.
